artman/cli/main.py

```diff
@@ -291,6 +299,14 @@ def normalize_flags(flags, user_config):
     pipeline_args['local_paths'] = support.parse_local_paths(
         user_config, flags.root_dir)
 
+    if flags.subcommand == 'publish' and flags.local_repo_dir:
+        if not flags.dry_run:
+            logger.error('`--dry-run` flag must be passed when '
+                         '`--local-repo-dir` is specified')
+            sys.exit(96)
```

Comment:
why 96? :)

Reply:
Artman convention :)

Reply:
For all error messages? It would be nice to have different codes for different errors to make scripting easier.

Reply:
The current convention is a bad practice. I will follow up with a PR to categorize those errors and return different error code.